example/src/main/scala/example/HelloWorldAdvanced.scala

```diff
@@ -24,22 +24,22 @@ object HelloWorldAdvanced extends App {
   private val server =
     Server.port(PORT) ++              // Setup port
       Server.paranoidLeakDetection ++ // Paranoid leak detection (affects performance)
-      Server.app(fooBar +++ app)      // Setup the Http app
+      Server.app(fooBar +++ app) ++   // Setup the Http app
+      Server.transport(Auto)          // (Server.epoll, Server.kqueue, Server.uring, Server.auto)
```

Comment:
```suggestion
      Server.transport(Transport.Auto)          // (Server.epoll, Server.kqueue, Server.uring, Server.auto)
```

Alternative API to do something like 

```scala
Server.nio
```